cmd/util/ledger/migrations/staged_contracts_migration.go

```diff
@@ -148,11 +153,131 @@ func (m *StagedContractsMigration) InitMigration(
 	return nil
 }
 
+// collectAndRegisterStagedContractsFromPayloads scan through the payloads and collects the contracts
+// staged through the `MigrationContractStaging` contract.
+func (m *StagedContractsMigration) collectAndRegisterStagedContractsFromPayloads(allPayloads []*ledger.Payload) error {
+	var stagingAccount string
+
+	// If the contracts are already passed as an input to the migration
+	// then no need to scan the storage.
+	// TODO: Maybe do this anyway?
+	if len(m.contractsByLocation) > 0 {
+		return nil
+	}
+
+	switch m.chainID {
+	case flow.Testnet:
+		stagingAccount = "0x2ceae959ed1a7e7a"
+	case flow.Mainnet:
+		// TODO:
+		stagingAccount = "0x2ceae959ed1a7e7a"
```

Comment:
This contract was just deployed today at [0x56100d46aa9b0212](https://contractbrowser.com/A.56100d46aa9b0212.MigrationContractStaging)

```suggestion
		stagingAccount = "0x56100d46aa9b0212"
```